src/datasets/splits.py

```diff
@@ -505,7 +505,7 @@ def __getitem__(self, slice_value):
         return split_instruction
 
     def get_list_sliced_split_info(self):
-        return list(sorted(self._splits.values(), key=lambda x: x.split_info.name))
+        return self._splits.values()
```

Comment:
I think this is not used anywhere else... But, just in case, better returning a list here?

Reply:
didn't find any usages either... applied your suggestion [6e10c33](https://github.com/huggingface/datasets/pull/5201/commits/6e10c33259d213f83ff92fb702b98f6fa3cffb07), thank you!